tests/models/nn/test_bert4rec.py

```diff
@@ -0,0 +1,538 @@
+import typing as tp
```

Comment:
To fix missing directory error during tests please set `enable_checkpointing=False` in `TransformerModelBase` default trainer.
Also we will use only 1 device by default (because changing this number is the first thing that we do everywhere). Also cross-validate will be correct to run only on one device.

Final default arguments for default trainer are here: https://github.com/MobileTeleSystems/RecTools/blob/976c14a72b9c71122ed406a9278c259c931e6969/rectools/models/nn/transformer_base.py#L598

Don't mind code logic change in my code. For now just change the arguments.